harvester_e2e_tests/conftest.py

```diff
@@ -49,9 +50,42 @@ def check_depends(self, depends, item):
 DepMgr.checkDepend, DepMgr._check_depend = check_depends, DepMgr.checkDepend
 
 
+def merge_config(user_config_path, default_config_path):
+    """
+    Merge two config files. The user config takes precedence over the default
+    config.
+    """
+
+    try:
+        with open(user_config_path, 'r') as ucnf:
+            user_data = yaml.safe_load(ucnf)
+    except FileNotFoundError:
+        user_data = {}
+
+    try:
+        with open(default_config_path, 'r') as dcnf:
+            default_data = yaml.safe_load(dcnf)
+    except FileNotFoundError:
+        default_data = {}
+
+    config = {}
+    for key in default_data:
+        config[key] = user_data.get(key, default_data[key])
+
+    for key in user_data:
+        if not key in default_data.keys():
+            config[key] = user_data[key]
+
```

Comment:
May be more pythonic?
```
>>> dcnf = {'A': 1, 'B': 2}
>>> ucnf = {'a': 10, 'B': 20, 'c': 30}
>>> ucnf | dcnf                         # python 3.9+
{'a': 10, 'B': 2, 'c': 30, 'A': 1}
>>> 
>>> {**ucnf, **dcnf}                    # python 3.5+
{'a': 10, 'B': 2, 'c': 30, 'A': 1}
```
Ref. https://stackoverflow.com/questions/38987/how-do-i-merge-two-dictionaries-in-a-single-expression-in-python